Gems/LmbrCentral/Code/Source/Shape/EditorCapsuleShapeComponent.cpp

```diff
@@ -116,9 +144,22 @@ namespace LmbrCentral
         ShapeComponentNotificationsBus::Event(
             GetEntityId(), &ShapeComponentNotificationsBus::Events::OnShapeChanged,
             ShapeComponentNotifications::ShapeChangeReasons::ShapeChanged);
+
+        AzToolsFramework::ComponentModeFramework::ComponentModeSystemRequestBus::Broadcast(
+            &AzToolsFramework::ComponentModeFramework::ComponentModeSystemRequests::Refresh,
+            AZ::EntityComponentIdPair(GetEntityId(), GetId()));
+
         return AZ::Edit::PropertyRefreshLevels::ValuesOnly;
     }
 
+    void EditorCapsuleShapeComponent::OnTransformChanged(
+        const AZ::Transform& /*local*/, const AZ::Transform& /*world*/)
```

Comment:
suggestion: We could potentially use `[[maybe_unused]]` here instead?

Reply:
oh yeah, that was from a copy paste, I'll update it.